src/components/CourseCard/CourseCard.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import {
+  Button,
+  Card,
+  CardContent,
+  Collapse,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { Course } from '@prisma/client';
+import { useState } from 'react';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+
+type Props = {
+  course: Course;
+};
+
+const CourseCard = ({ course }: Props) => {
+  const theme = useTheme();
+  const [expanded, setExpanded] = useState(false);
+
+  return (
+    <Card
+      sx={{
+        background: theme.palette.darkBlue.main,
+        color: theme.palette.white.main,
+        width: 340,
+        maxWidth: '100%',
+        borderRadius: '15px',
+        margin: 'auto',
+        textAlign: 'center',
+      }}
+    >
+      <CardContent sx={{ overflowWrap: 'break-word' }}>
+        <Typography variant="h5" m={2}>
+          {course.name}
+        </Typography>
+        <Typography>{course.startDate.toLocaleString()}</Typography>
```

Comment:
Browser console gives error and switches to client side rendering as client and server locales don't match. Could be resolved with NEXT_LOCALE cookie, but could be also fixed using other than localized time string.  
```suggestion
        <Typography>{course.startDate.toDateString()}</Typography>
```

Reply:
I went with the `toDateString`. I also noticed that there is a `NoSsr` Material UI component that could work if we wanted to prevent server side rendering on some elements:
https://mui.com/material-ui/react-no-ssr/